### ISSUE TITLE
Set strategy to binpack | Swarm 1.12.1

### ISSUE THREAD

**Issue by pascalandy** [docker/docker#26700]
I understand Docker Swarm is using strategy spread by default.
https://docs.docker.com/swarm/scheduler/strategy/

How can I configure the strategy=binpack? I don't see a way to update this behaviour.

Thank you!

Server:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 05:33:38 2016
 OS/Arch:      linux/amd64

**Comment by thaJeztah** [comment]
more strategies will be added in future (see https://github.com/docker/swarmkit/issues/1034, and https://github.com/docker/swarmkit/issues/308) , but not yet available at this point.

/cc @aluzzardi

**Comment by pascalandy** [comment]
Thanks!

**Comment by thaJeztah** [comment]
I'll keep it open for now, as others may have this question as well, but perhaps we can create a "tracking" issue with more information about other strategies that are planned once that information is available.

**Comment by thaJeztah** [comment]
We discussed this feature request; implementing this requires more discussion, and it's probably best to track this through the linked issues in Swarmkit.

TL;DR bin pack doing the right way is non-trivial, and _if_ we implement it, we should avoid having the same issues as "classic" Swarm has.

I'll close this issue here for housekeeping, but thanks again for suggesting!
